_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -168,7 +168,8 @@ mfxStatus SetRateControl(
     VADisplay    vaDisplay,
     VAContextID  vaContextEncode,
     VABufferID & rateParamBuf_id,
-    bool         isBrcResetRequired = false)
+    bool         isBrcResetRequired = false,
+    bool         isBrcSlidingWindowFrameToleranceMode = false)
```

Comment:
I think it's better to pass m_caps to the function instead of this parameter with huge name.